src/components/contacts/ContactRow.js

```diff
@@ -58,17 +59,20 @@ const ContactRow = ({ address, color, nickname, symmetricalMargins, ...props },
   const profilesEnabled = useExperimentalFlag(PROFILES);
   const { width: deviceWidth } = useDimensions();
   const { onAddOrUpdateContacts } = useContacts();
-  const { nativeCurrency } = useAccountSettings();
   const { colors } = useTheme();
-  const { accountType, balance, ens, image, label, network, onPress, showcaseItem, testID } = props;
+  const { accountType, ens, image, label, network, onPress, showcaseItem, testID } = props;
```

Comment:
leaving a note to check if this is actually what we want for ALL instances of ContactRow vs the instances that had `balance` passed in (presumably your own wallets that show up in the Send sheet)